package.json

```diff
@@ -26,7 +26,7 @@
     "test:eslint": "eslint --config ./.eslintrc.yaml \"source/js/**/*.js\" \"source/js/components/**/*.jsx\" webpack.config.js",
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/components/**/*.scss\" --syntax scss",
     "test": "npm run build && run-s test:**",
-    "watch:images": "chokidar 'source/images/**/*' -c 'npm run build:images'",
+    "watch:images": "chokidar \"source/images/**/*\" -c \"npm run build:images\"",
```

Comment:
I see you're making it consistent with the rest of the file, but...what is going on here?